README.md

```diff
@@ -17,46 +17,45 @@ The `MobileCore` and `Identity` extensions represent the foundation of the Adobe
 
 `Lifecycle` represents the Adobe Experience Platform SDK's `Lifecycle` extension that helps collect application Lifecycle metrics, such as application install or upgrade information, application launch and session information, device information, and any additional context data provided by the application developer. 
 
-## Requirements
+## Getting started
 
-- Android API 19 (Kitkat) and newer.
-
-## Installing the AEP SDK for Android
-
-Installation via [Maven](https://maven.apache.org/) & [Gradle](https://gradle.org/) is the easiest and recommended way to get the AEP SDK into your Android app.  In your `build.gradle` file, include the latest version of following dependencies:
-
-> The following 2.x.x should be replaced with the latest version numbers.
-
-```gradle
-implementation 'com.adobe.marketing.mobile:core:2.x.x'
-implementation 'com.adobe.marketing.mobile:identity:2.x.x'
-implementation 'com.adobe.marketing.mobile:signal:2.x.x'
-implementation 'com.adobe.marketing.mobile:lifecycle:2.x.x'
-```
+Refer to the [getting started](./Documentation/GettingStarted.md) guide for setting up and using the SDK with your application.
 
 ## Development
 
 ***Open the project*** To open and run the project, open the `code/build.gradle` file in Android Studio
 
 ***Run demo application*** Once you open the project in Android Studio (see above), select the `testapp` or `testapp-kotlin` runnable and your favorite emulator and run the program.
 
-## Related Projects
+## Documentation
 
-| Project                                                      | Description                                                  |
-| ------------------------------------------------------------ | ------------------------------------------------------------ |
-| [AEP SDK Sample App for Android](https://github.com/adobe/aepsdk-sample-app-android) | Contains Android sample app for the AEP SDK. |
-| [AEP Edge SDK for Android](https://github.com/adobe/aepsdk-edge-android) | Provides support to the Experience Platform Edge for the AEP SDK.|
- | [AEP Assurance SDK for Android](https://github.com/adobe/aepsdk-assurance-android) |  Provides support to inspect and validate AEP SDK by integrating with Adobe Experience Platform Assurance. |
+Additional documentation about API usage and SDK architecture can be found under the [Documentation](./Documentation) directory.
 
-## Documentation
+## Related Projects
 
-Additional documentation for usage and SDK architecture can be found under the [Documentation](Documentation) directory.
+| Project | Maven | Github |
+|---|---|---|
+| [Profile](https://developer.adobe.com/client-sdks/documentation/profile/) | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/userprofile.svg?logo=android&logoColor=white&label=userprofile&style=flat-square)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/userprofile) | [Link](https://github.com/adobe/aepsdk-userprofile-android) |
```

Comment:
Can we link just the github projects in this table and remove the docs one? Also some projects provide a short description for each extension, e.g.
| [Analytics extension](https://github.com/adobe/aepsdk-analytics-android)             | The Analytics extension allows for sending mobile interaction data from your app to Adobe Analytics. |
| [Assurance extension](https://github.com/adobe/aepsdk-assurance-android)             | The Assurance extension enables validation workflows for your SDK implementation.                    |